servers/src/test/java/tachyon/master/file/meta/InodeTreeTest.java

```diff
@@ -79,7 +83,13 @@ public void before() throws IOException {
     mTree = new InodeTree(blockMaster, directoryIdGenerator, mountTable);
 
     blockMaster.start(true);
+
+    TachyonConf conf = new TachyonConf();
+    conf.set(Constants.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "true");
+    conf.set(Constants.SECURITY_AUTHORIZATION_PERMISSION_SUPERGROUP, "superGroup");
```

Comment:
replacing the default supergroup with superGroup is subtle ... let's use test-supergroup instead